package.json

```diff
@@ -18,11 +18,5 @@
   },
   "license": "FACT-Finder",
   "homepage": "http://web-components.fact-finder.de/",
-  "private": true,
-  "ignore": [
```

Comment:
It seems someone tried to do something similar to bower.json. Or maybe it's just some remnant.